### DISCUSSION TITLE
Windows Defender flagged Trojan:Win32/Wacatac.B!ml — Anyone else?

### DISCUSSION THREAD

**Discussion by OhhNico** [winstxnhdw/lc-hax#572] [General]
Hi everyone,

I recently downloaded the lc-hax repository and tried to use it. However, my Windows Defender immediately flagged one of the files as a Trojan (Trojan:Win32/Wacatac.B!ml) and blocked it.

I'm curious if anyone else using Windows Defender has encountered the same issue, or if this might be specific to my system. It would be great to know if this is a false positive or if there's something else going on.

Has anyone else experienced this?

Thanks for your feedback!

**Comment by winstxnhdw** [comment]
Windows will sometimes flag the injector as a trojan, probably because it literally manipulates Unity's memory. I have documented how you can get around this in the README [here](https://github.com/winstxnhdw/lc-hax?tab=readme-ov-file#whitelisting-sharpmonoinjectorcore). There's also a bunch of pre-existing discussions/issues on this topic.

**Reply by OhhNico** [reply]
ty so much!
